### PR TITLE
DE43306 Fix for My CPD date filter 

### PR DESCRIPTION
**Pull request by Syeda-Azim**
https://rally1.rallydev.com/#/431372673576d/backlog?detail=%2Fdefect%2F601113019692

### CODE REVIEW

File: src/components/cpd-my-records.js [hunk 1]
```diff
@@ -210,11 +210,12 @@ class MyCpdRecords extends BaseMixin(LitElement) {
 	}
 
 	getUtcDate(dateStr) {
-		const localDate = new Date(dateStr);
-		if (localDate === undefined || localDate === null)
-			return localDate;
+		if (dateStr === undefined || dateStr === null || dateStr === '')
+			return dateStr;
```
Comment (imcdonald-d2l) [inline src/components/cpd-my-records.js:214]: Does the date filter work with `undefined`, `null` and `''` values? A gif of this working would be appreciated. I use this: https://www.screentogif.com/

File: src/components/cpd-my-records.js [hunk 2]
```diff
@@ -210,11 +210,12 @@ class MyCpdRecords extends BaseMixin(LitElement) {
 	}
 
 	getUtcDate(dateStr) {
-		const localDate = new Date(dateStr);
-		if (localDate === undefined || localDate === null)
-			return localDate;
+		if (dateStr === undefined || dateStr === null || dateStr === '')
```
Comment (IhabAlkabary) [inline src/components/cpd-my-records.js:213]: wouldn't this do the same thing?
```
if( !dateStr ) {
    return dateStr;
}
```

**Comment by IhabAlkabary** [comment]
Rally story link in the description is for a different story

File: src/components/cpd-my-records.js (hunk 1 above)
Comment (Syeda-Azim) [reply src/components/cpd-my-records.js:214]: https://user-images.githubusercontent.com/62559837/116723165-32b39b80-a9ad-11eb-9641-7d26edd8d1cc.mp4

**Comment by Syeda-Azim** [comment]
> Rally story link in the description is for a different story

My bad! link is updated.

**Review by imcdonald-d2l — APPROVED** [review]

**Pull request merged**